tests/test_io.py

```diff
@@ -1898,8 +1900,59 @@ def test_get_variables_data_values4(self):
         }
 
         for index, test_data in test_data_sets.items():
-            np.testing.assert_array_almost_equal(test_data[0], reference_data['time'][index])
-            np.testing.assert_array_almost_equal(test_data[1], reference_data['@Diagnostics.nbr_steps'][index])
+            np.testing.assert_array_almost_equal(test_data['time'].x, reference_data['time'][index])
+            np.testing.assert_array_almost_equal(test_data['@Diagnostics.nbr_steps'].x, reference_data['@Diagnostics.nbr_steps'][index])
+
+    @testattr(stddist = True)
```

Comment:
I added these tests as there were reports of start/stop not properly being accounted for.
However when I test this locally it works all fine for me, the reporter will come back next week after investigating if there was a user error.